js/collision.js

```diff
@@ -1,64 +0,0 @@
-var collisionDetection = {
-    collision: function () {
-        collisionDetection.checkCollisionInterval = setInterval(function () {
-            if (collisionDetection.checkCollisionPosition($('.obCar1'), $('#myCar'))) {
-                collisionDetection.stopAllInterval();
```

Comment:
collisionDetection.stopAllInterval(); is common across all the if else blocks, it should be common and called only once